datalad/cmd.py

```diff
@@ -200,7 +205,7 @@ class SafeDelCloseMixin(object):
     def __del__(self):
         try:
             self.close()
-        except TypeError:
+        except (TypeError, ImportError):
```

Comment:
could you leave a comment with when `ImportError` (and `TypeError` if you remember) is expected to happen?  not clear to me why only those 2.

Reply:
Not sure about `TypeError`, the code existed before, but I saw `ImportError`, if `BatchedCommand`-instances are deleted when the python interpreter is exiting. Comment added.